source/code-snippets/usage-examples/findOne.ts

```diff
@@ -0,0 +1,40 @@
+import { FindOptions, MongoClient } from "mongodb";
```

Comment:
```suggestion
import { MongoClient } from "mongodb";
```

If we go with my other suggestion of inlining the usage.